config/core/resources/cloudbuildsource.yaml

```diff
@@ -54,6 +60,9 @@ spec:
     - name: v1alpha1
       served: true
       storage: true
+    - name: v1beta1
+      served: true
+      storage: false
```

Comment:
@Harwayne shall we change everywhere to?

  versions:
  - name: v1alpha1
    served: false
    storage: false
  - name: v1beta1
    served: true
    storage: true

Reply:
I don't think I can change it to things as above.
Otherwise, we can't create v1alpha1 resources anymore
It looks like it should be 
  versions:
  - name: v1alpha1
    served: true
    storage: false
  - name: v1beta1
    served: true
    storage: true

Reply:
ok, please sync with Adam tomorrow, as he knows better about this... If it passes, I think we can let it in...